packages/components/src/internal/components/ontology/OntologyConceptPicker.tsx

```diff
@@ -35,7 +35,12 @@ export const OntologyConceptPicker: FC<Props> = memo((props: Props) => {
 
     useEffect(() => {
         if (conceptSubtree) {
-            fetchPathModel(conceptSubtree).then(setSubtreePath);
+            fetchPathModel(conceptSubtree)
+                .then(setSubtreePath)
+                .catch(() => {
+                    // using null to signal that loading is complete
+                    setSubtreePath(null);
```

Comment:
Probably works, but seems a nonstandard way to do this signaling that then necessitates the comment.  Is there a reason we don't want to use an `isLoading` or `isLoaded` flag?

Reply:
fixed, added `isLoadingSubtreePath`